swarm/network/protocol.go

```diff
@@ -243,7 +242,7 @@ func (self *bzzPeer) LastActive() time.Time {
  Handshake
 
 * Version: 8 byte integer version of the protocol
-* NetworkID: 8 byte integer network identifier
+* NetworkId: 8 byte integer network identifier
```

Comment:
So initially changed `networkId` to `networkID` but then realised all other code (e.g. in `eth`) uses `networkId` so I switched it back, but missed the fact that this was already `networkID`.

Should I just change the swarm code to `networkID`?